### ISSUE TITLE
1.3.0.rc1 rake solidus:upgrade:one_point_three fails

### ISSUE THREAD

**Issue by jrochkind** [solidusio/solidus#1245]
~~~
$ ./bin/rake solidus:upgrade:one_point_three
rake aborted!
Don't know how to build task 'solidus:migrations:create_vat_prices' (see --tasks)
Tasks: TOP => solidus:upgrade:one_point_three
(See full trace by running task with --trace)
$
~~~

Mystery to me too, it looks like that rake task is right there... 
https://github.com/solidusio/solidus/blob/v1.3.0.rc1/core/lib/tasks/migrations/create_vat_prices.rake

Haven't had a chance to try and debug it myself yet.

**Comment by jordan-brough** [comment]
I've opened a PR for this: #1246.

**Comment by jhawthorn** [comment]
This should be fixed by #1246

**Comment by jrochkind** [comment]
Thanks! It's still unclear to me whether I really need to run the rake task or not -- the release notes now only mention it in passing in one place about historical tax data for shipping.  My app _appears_ to be running fine without having run it. Can the release notes be made more clear about how strong the requirement or recommendation for the rake task is, what it actually is gonna do, and if I'm making a mistake in skipping it? :)
